### DISCUSSION TITLE
Downgrading

### DISCUSSION THREAD

**Discussion by Lytonicity** [3kh0/ext-remover#1034] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does downgrading a Chromebook also enroll the Chromebook? (and will my admin know)?

**Comment by AshtonDavies** [comment, marked as answer]
Downgrading does not unenroll the Chromebook if that's what you meant. The process itself does not notify your administrator. What they can see includes usage data, such as sign-ins and the time of last activity.

![gac.fcd151d7_Z1lgEXy.png](https://github.com/3kh0/ext-remover/assets/117958718/1a8c5b2d-ce05-4ca4-ae0c-9a085b2c2881)

**Reply by Lytonicity** [reply]
ok thx dude
